### ISSUE TITLE
How to use themeing/styling?

### ISSUE THREAD

**Issue by DennisKuhn** [SharePoint/sp-dev-docs#834]
#### Category
- [x] Question
- [ ] Typo
- [ ] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
One consistent way to apply themeing.

#### Observed Behavior
I am confused how to integrate the themeing.
It seems to me that colours are included via `.scss`, as in the [samples/react-themes](https://github.com/SharePoint/sp-dev-fx-webparts/tree/master/samples/react-themes) and [StfBauers](https://github.com/StfBauer) - [Theme Token Variables/#761](https://github.com/SharePoint/sp-dev-docs/issues/761) .
Font classes are imported in the `.tsx` file `import { FontClassNames } from '@uifabric/styling';` and used like `FontClassNames.mediumPlus`. Is that correct by using ?
In `ColorClassNames` imported from `@uifabric/styling` I found all colours but primaryText. Should I use `themePrimary` or `neutralPrimary` instead?

#### Steps to Reproduce
test.scss:
```
$ms-primaryBackground:          "[theme:primaryBackground, default:#ffffff]";
$ms-primaryText:                "[theme:primaryText, default:#333333]";
$ms-themePrimary:               "[theme:themePrimary, default:#0078d7]";

.test {
    background-color: $ms-primaryBackground;
    color: $ms-primaryText;

    .heading {
        color: $ms-themePrimary;
    }
}
```
test.tsx:
```
import {
    FontClassNames
} from '@uifabric/styling';
...

const headingStyles = [styles.heading, FontClassNames.mediumPlus];
const contentStyles = [FontClassNames.medium];

public render(): JSX.Element {
    return ( <div className={css( styles.test ) }>
        <div className={ css( ... headingStyles ) }>Heading 1</div>
        <div className={ css( ... contentStyles ) }>Content 1 text</div>
        <div className={ css(  ... headingStyles  ) }>Heading 2</div>
        <div className={ css( ... headingStyles ) }>Content 2 text</div>
    </div>);
}
```

Is this the recommended/right way to do it? The Office-UI-Fabric styling references weight and size classes as well. How  are they supposed to be included? 

Bottom line, I wonder how to to integrate the office styling proper in one place, so I can simply define my style based on the standard style. E.g.:

```
NormalHeading: ColorClassNames.ThemePrimary, FontClassNames.medium, FontWeight.semibold
NormalText: ColorClassNames.PrimaryText, FontClassNames.medium
SmallHeading: ColorClassNames.ThemePrimary, FontClassNames.small, FontWeight.semibold
SmallText: ColorClassNames.PrimaryText, FontClassNames.small
```

**Comment by VesaJuvonen** [comment]
This issues should be fully solved and fixed, given the work we have done in engineering in this area. To ensure that we clean up the backlog of the issues, we will be closing this one. If you still have issues on this particular area, please open up a new issue so that we can get updated issue description and reproduce steps. We do apologize the non-optimal handling of these old issues and are looking into improving this in future.
